lighthouse-core/lib/i18n/i18n.js

```diff
@@ -56,6 +56,8 @@ const UIStrings = {
   columnWastedMs: 'Potential Savings',
   /** Label for the time spent column in data tables, entries will be the number of milliseconds spent during a particular activity */
   columnTimeSpent: 'Time Spent',
+  /** Label for the location column in data tables, entries will be the location of a specific entity, the entity may be a file in a directory, or a place in the world, it is generic */
```

Comment:
maybe switch to the `Label for a column in a data table;` opening instead of this version. But more importantly

> entries will be the location of a specific entity, the entity may be a file in a directory, or a place in the world, it is generic

I am not a translation expert, but this sounds like it could be a problem for translating :)

Are all uses for violation audits, though? I think in all of those this refers to a line number (in the form `line: 525`), then, and maybe we should pick a new name for the variable, like `columnLocationLineNumber` (so good)...or rename the table header label?

Reply:
Yup, I was mistaken that it is a generic!